package.json

```diff
@@ -21,10 +21,13 @@
     "@protobuf-ts/grpc-transport": "^2.9.0",
     "@protobuf-ts/runtime-rpc": "^2.9.0",
     "@swc/helpers": "0.5.3",
+    "@types/imurmurhash": "^0.1.3",
```

Comment:
Should `imurmurhash` also be a dependency?

Reply:
Adde to module as a dependency